HACKING.rst

```diff
@@ -31,7 +31,7 @@ you may need to run::
 
   pip install importlib
 
-You may get an error 'psycopg2 package not found' when you do 'haas init_db' 
+You may get an error 'psycopg2 package not found' when you do 'haas-admin db create'
```

Comment:
The `db` subcommand has a bunch of stuff in it provided by the migration framework (including upgrades); it seemed the place to add it. The `create` subcommand does what `haas init_db` did before. I'm actually not sure I think this is the right thing; most (but not all) of the other stuff under `haas-admin db` is developer-oriented, so it might make sense to have another entry point to the create/upgrade commands. I called it `create` because `init` already does something else, but that's part of what makes me think I should just move it to another command; `init` is really only interesting to devs.

The framework provides some of it's own CLI tooling, so I put this in a separate command since (1) we've been talking about splitting out the client/server stuff into two or more commands anyway, and (2) it meant not having to write as much glue code.